lang/en/lang.php

```diff
@@ -1,55 +1,55 @@
 <?php
 
-$lang['nocookies']       = "You don't have pages selected yet. Check your browser to allow cookies and try again.";
-$lang['empty']           = "You don't have pages selected yet.";
-$lang['toprint']         = "Selected pages";
-$lang['removed']         = "Pages removed from selection";
+$lang['nocookies']       = "You don't have any pages selected yet. Check your browser to allow cookies and try again.";
+$lang['empty']           = "You don't have any pages selected yet.";
+$lang['toprint']         = "Selected Pages";
+$lang['removed']         = "Pages Removed from Selection";
 
-$lang['reserconfirm']    = "Do you really clear selection?";
-$lang['reset']           = "Clear selection";
+$lang['reserconfirm']    = "Do you really want to clear the selection?";
+$lang['reset']           = "Clear Selection";
 
 $lang['js']['include']   = "Add to selection";
 $lang['js']['remove']    = "Remove page from selection";
-$lang['export']          = "Export selection";
+$lang['export']          = "Export Selection";
 $lang['create']          = "Export";
 
 $lang['exportpdf']       = "Export selection to PDF";
 $lang['exportodt']       = "Export selection to ODT";
 $lang['exportprint']     = "Printable version";
 $lang['exporttext']      = "Text only";
 
-$lang['title']           = "Document's title (optional)";
+$lang['title']           = "Document's Title (optional)";
```

Comment:
Hmm, I'm not sure if I like these capitals. Why did you capitalize the second words in these strings?